Density/app/src/main/java/org/cornelldti/density/density/data/OperatingHoursClass.kt

```diff
@@ -0,0 +1,10 @@
+package org.cornelldti.density.density.data
+
+class OperatingHoursClass(todayOperatingHours: List<Pair<Long, Long>>, tomorrowFirstOpHours: Pair<Long, Long>) {
```

Comment:
I feel like a Kotlin `data class` would be perfect for this: https://kotlinlang.org/docs/reference/data-classes.html